File: HelloWorld.java
import java.util.Scanner;

public class HelloWorld {
    public static void main(String[] args) {
        System.out.println("Hello, World!");
        Scanner scanner = new Scanner(System.in);
        System.out.print("Введите ваше имя: ");
        String name = scanner.nextLine();
        System.out.println("Привет, " + name + "!");
        scanner.close();
    }
}

File: HelloWorld.c
// hello.c
#include <stdio.h>

int main() {
    printf("Hello, World!\n");

    char name[50];
    printf("Введите ваше имя: ");
    scanf("%49s", name);
    printf("Привет, %s!\n", name)
    return 0;
}